src/components/SelectFormField/styles.module.scss

```
@@ -1,70 +1,90 @@
     composes: item from '../FormFields/styles.module.scss';
+}
 
-    :global {
-        .ant-select-selector {
-            min-height: 48px !important;
-            padding: 4px 8px !important;
-            border: 1px solid var(--input-border-color) !important;
-            background: var(--input-bg);
-            box-shadow: none !important;
-        }
+.select :global(.ant-select-selector) {
```

Stylelint (17.13.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)